priv/templates/nimble_template/assets/js/helpers/storageAvailable.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * @see https://developer.mozilla.org/en-US/docs/Web/API/Web_Storage_API/Using_the_Web_Storage_API
+ */
+
+const STORAGE_TYPES = {
+  localStorage: "localStorage",
+};
+
+function isStorageAvailable(type) {
```

Comment:
Not sure about this, are you seeing us using this a lot 🤔 ?

Reply:
so we can remove it, right :D

Reply:
Removed in https://github.com/nimblehq/elixir-templates/pull/154/commits/ee7be53e5dbe91a94179262de72b85db22e61cb3